.gitignore

```diff
@@ -8,5 +8,6 @@ converted.pdf
 /public
 .firebase/
 firebase-debug.log
+.idea/
```

Comment:
Do not include this

Reply:
.gitgnore just ignores files to not include them when we do `git add .`
So, that won't affect anything